lib/utils/storage.ts

```diff
@@ -0,0 +1,61 @@
+import * as azure from "azure-storage";
+import { Option, option } from "ts-option";
+import { Either, left, right } from "./either";
+
+const STORAGE_CONNECTION_STRING = process.env.AzureWebJobsStorage;
+
+const blobService = azure.createBlobService(STORAGE_CONNECTION_STRING);
```

Comment:
you should never instantiate types (esp. services) in utility modules, let the client do that and pass the instance to the utility